_posts/events/2018-08-28-propublica-parking-tickets.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: event
+categories:
+  - events
+links:
+title: "Parking Tickets and Bankruptcy"
+description: "If you're a Chicagoan, Jason Kunesh is your design director. What does that even mean? He's working to introduce product & service design disciplines and techniques to make sure the city is accessible, inclusive, and strategic when it launches tech products."
+speakers: Melissa Sanchez, ProPublica Illinois reporter, and David Eads, ProPublica Illinois news apps developer
```

Comment:
This can be formatted as a list, like so:

```
speakers:
 - Melissa Sanchez, ProPublica Illinois reporter
 - David Eads, ProPublica Illinois news apps developer
```